static/app/components/events/contexts/trace/getTraceKnownDataDetails.tsx

```diff
@@ -17,8 +17,8 @@ type Output = {
 type Props = {
   data: TraceKnownData;
   event: Event;
-  organization: Organization;
   type: TraceKnownDataType;
+  organization?: Organization;
```

Comment:
Is the org really not required? I'd be careful making these optional since it's easy to forget to pass it even if you have access to it

Reply:
Yeah, this is a bit challenging. I can take a look at this again but you make a good point. The reason I opted to make these optional was so that invoking that unused function `getFormattedContextData`, you wouldn't need to pass in extra props if they weren't necessary. For example, `AppContext`doesn't need organization, but `TraceContext` does, so you could add those additional props only for retrieving context that needs it.

That does make a problem where it expects the caller to know how the internals work already, and what their context needs to resolve. Instead, I've made a change in 6a302ca30a to require `organization` to be passed in regardless of whether or not it gets used (currently, only `ProfileContext` and `TraceContext`, to get around this issue. `project` is still optional, because `ProfileContext` marked it as optional from the get go.